.github/workflows/release-published.yml

```diff
@@ -8,4 +8,6 @@ on:
 jobs:
   release:
     uses: liquibase/build-logic/.github/workflows/extension-release-published.yml@v0.7.8
```

Comment:
@jandroav @sayaliM0412 if we are merging the new input into the `main` branch, do we need to change line 10 to be @main ?

Reply:
I was thinking of re-releasing `v0.7.8` with these changes.

Reply:
I just changed it to using @main